kuma/static/js/task-completion.js

```diff
@@ -17,7 +17,9 @@
     function inquire() {
 
         // dimension14 is "Saw Survey Gizmo Task Completion survey"
-        if(win.ga) ga('set', 'dimension14', 'Yes');
+        if(win.ga) {
+            ga('set', 'dimension14', 'Yes');
+        }
```

Comment:
Should the next line (``ga('send'...``) also be in this block?

Reply:
Hm. It would not have been part of the same if block in the original code. So this PR does not change that functionality... but you pose a valid question.

Reply:
I settled on removing the if statement altogether. It's not around the other ga() calls on the page and not necessary.

The ga object is there even if the user blocks the script from loading the rest of the google code.

Reply:
I think the exception is in development mode, when assets are not combined, and the developer's ad blocker blocks ``analytics.js``.  Not needed to protect production, and developers can diagnose and fix.